lua/rustaceanvim/lsp/init.lua

```diff
@@ -354,10 +344,9 @@ end
 
 ---Restart the LSP client.
 ---Fails silently if the buffer's filetype is not one of the filetypes specified in the config.
----@param bufnr? number The buffer number (optional), defaults to the current buffer
```

Comment:
question: is there a reason for removing this parameter? It seems out of scope for this PR.

Reply:
the param seems obsolete and wasn't actually used or set